Easydict/App/Localizable.xcstrings

```diff
@@ -3834,6 +3850,16 @@
         }
       }
     },
+    "summary_service" : {
+      "localizations" : {
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "总结"
```

Comment:
Looks like this missing the `en` localization.

<img width="337" alt="Screenshot 2024-07-14 at 11 32 29" src="https://github.com/user-attachments/assets/f0b55b78-88d4-4ff4-85af-368beb1cb05f">

Reply:
My bad, thank you

Reply:
Sorry I didn’t take my laptop with me. Please let me fix it tonight

Reply:
I have added this en localization.

Reply:
OK, it's my bad, I thought it has been fixed and not check the commit. 🥲